projects/driver_display/inc/driver_display_brightness.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+// Module used to control the brightness of the driver display screens.
+// Also defines display screen parameters such as frequency and number of screens
+
+#include "adc.h"
+#include "gpio.h"
+#include "pwm.h"
+#include "soft_timer.h"
+
+#define DRIVER_DISPLAY_CONFIG_NUM_SCREENS 2
+
+#define DRIVER_DISPLAY_CONFIG_SCREEN1_FREQ_HZ 30000  // Frequency in Hz (subject to change)
+#define DRIVER_DISPLAY_CONFIG_SCREEN2_FREQ_HZ 30000  // Frequency in Hz (subject to change)
+
+#define DRIVER_DISPLAY_CONFIG_REFRESH_PERIOD 5
+
+typedef enum {
+  DRIVER_DISPLAY_SCREEN1 = 0,
+  DRIVER_DISPLAY_SCREEN2,
+} DriverDisplayBrightnessScreen;
+
+typedef struct {
+  GPIOAddress address;
+  GPIOSettings settings;
+  PWMTimer timer;
+  uint16_t frequency_hz;  // frequency in Hz
+} DriverDisplayBrightnessScreenData;
+
+typedef struct {
+  GPIOAddress address;
+  GPIOSettings settings;
+  ADCChannel channel;
+  uint16_t max;
+  uint16_t min;
+  uint16_t percent_reading;
+} DriverDisplayBrightnessSensorData;
+
+typedef struct {
+  DriverDisplayBrightnessScreenData *screen_data;
+  DriverDisplayBrightnessSensorData sensor_data;
+} DriverDisplayBrightnessData;
```

Comment:
This doesn't follow our convention. I don't know how to use your module from looking at this. What do I configure? What needs to persist? What should be initialized beforehand?